### PR TITLE
[ACR] Remove hard-coded SDK import

### PR DESCRIPTION
**Pull request by djyou**
This PR partially fixes https://github.com/Azure/azure-cli/issues/8265

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by djyou** [comment]
@shahzzzam for `task`/`run`/`build` commands.
@NatalieYuGong for `import` command.
@northtyphoon for removing `build-task` commands.
/cc @sajayantony

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/run.py [hunk 1]
```diff
@@ -9,10 +9,7 @@
 from knack.log import get_logger
 from knack.util import CLIError
 from azure.cli.core.commands import LongRunningOperation
-from azure.mgmt.containerregistry.v2018_09_01.models import (
-    FileTaskRunRequest,
-    PlatformProperties,
-)
+
```
Comment (shahzzzam) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/run.py]: is this necessary?

**Comment by djyou** [comment]
@tjprescott

File: src/azure-cli-core/azure/cli/core/profiles/_shared.py [hunk 2]
```diff
@@ -96,16 +97,12 @@ def default_api_version(self):
...
-            'classic_administrators': '2015-06-01'
-        }),
-        ResourceType.DATA_STORAGE: '2018-03-28',
-        ResourceType.DATA_COSMOS_TABLE: '2017-04-17',
         ResourceType.MGMT_NETWORK_DNS: '2018-05-01',
         ResourceType.MGMT_KEYVAULT: '2018-02-14',
         ResourceType.MGMT_AUTHORIZATION: SDKProfile('2018-01-01-preview', {
             'classic_administrators': '2015-06-01'
         }),
+        ResourceType.MGMT_CONTAINERREGISTRY: '2018-09-01',
```
Comment (tjprescott) [inline src/azure-cli-core/azure/cli/core/profiles/_shared.py]: https://github.com/Azure/azure-rest-api-specs/tree/master/profile

ACR isn't part of either of these profiles. It should only appear in latest. Does it fail if you leave these out?

File: src/command_modules/azure-cli-acr/HISTORY.rst [hunk 3]
```diff
@@ -7,6 +7,8 @@ Release History
 ++++++
 * Add support for VNet/IP rules.
 * Add --platform parameter for `acr build`, `acr task create` and `acr task update` commands
+* BREAKING CHANGE: Remove 'acr build-task' command group.
```
Comment (tjprescott) [inline src/command_modules/azure-cli-acr/HISTORY.rst]: If there are breaking changes, this should be at least version 2.2.0

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py [hunk 4]
```diff
@@ -124,17 +98,13 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
...
-
     with self.argument_context('acr repository untag') as c:
         c.argument('image', arg_type=image_by_tag_type)
 
     with self.argument_context('acr create') as c:
         c.argument('registry_name', completer=None)
-        c.argument('deployment_name', arg_type=deployment_name_type, validator=None)
-        c.argument('location', arg_type=get_location_type(self.cli_ctx), validator=get_default_location_from_resource_group)
+        c.argument('deployment_name', validator=None)
+        c.argument('location', validator=get_default_location_from_resource_group)
```
Comment (tjprescott) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py]: Why did you remove these?  These still seem quite applicable.

File: src/azure-cli-core/azure/cli/core/profiles/_shared.py (hunk 2 above)
Comment (djyou) [reply src/azure-cli-core/azure/cli/core/profiles/_shared.py]: The CI was complaining couldn't find acr from these profiles.

Comment (tjprescott) [reply src/azure-cli-core/azure/cli/core/profiles/_shared.py]: Hmmm. Okay. I will create an issue for us to look into this.

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py (hunk 4 above)
Comment (djyou) [reply src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py]: As requested in previous PRs, they are already registered as global params https://github.com/Azure/azure-cli/blob/f4a60fb3d0ab05ef9cbbb54e2999e80206d0d3a9/src/azure-cli-core/azure/cli/core/__init__.py#L250

**Comment by djyou** [comment]
@tjprescott Ready to merge, thanks!
